render/wgpu/shaders/blend/multiply.wgsl

```diff
@@ -0,0 +1,52 @@
+#import common
+
+struct VertexOutput {
+    @builtin(position) position: vec4<f32>,
+    @location(0) uv: vec2<f32>,
+};
+
+#if use_push_constants == true
+    var<push_constant> transforms: common::Transforms;
+    @group(1) @binding(0) var parent_texture: texture_2d<f32>;
+    @group(1) @binding(1) var current_texture: texture_2d<f32>;
+    @group(1) @binding(2) var texture_sampler: sampler;
+#else
+    @group(1) @binding(0) var<uniform> transforms: common::Transforms;
+    @group(2) @binding(0) var parent_texture: texture_2d<f32>;
+    @group(2) @binding(1) var current_texture: texture_2d<f32>;
+    @group(2) @binding(2) var texture_sampler: sampler;
+#endif
+
+@vertex
+fn main_vertex(in: common::VertexInput) -> VertexOutput {
+    let pos = common::globals.view_matrix * transforms.world_matrix * vec4<f32>(in.position.x, in.position.y, 1.0, 1.0);
+    let uv = vec2<f32>((pos.x + 1.0) / 2.0, -((pos.y - 1.0) / 2.0));
+    return VertexOutput(pos, uv);
+}
+
+fn blend_func(src: vec3<f32>, dst: vec3<f32>) -> vec3<f32> {
+    return src * dst;
+}
+
+@fragment
+fn main_fragment(in: VertexOutput) -> @location(0) vec4<f32> {
+    // dst is the parent pixel we're blending onto
+    var dst: vec4<f32> = textureSample(parent_texture, texture_sampler, in.uv);
+    // src is the pixel that we want to apply
+    var src: vec4<f32> = textureSample(current_texture, texture_sampler, in.uv);
+
+    if (src.a > 0.0) {
```

Comment:
Could you add a comment here about the special-case in Flash?

Reply:
Added!